src/Phansible/Resources/ansible/roles/xdebug/tasks/main.yml

```diff
@@ -2,3 +2,11 @@
 - name: Install xDebug
   sudo: yes
   apt: pkg=php5-xdebug state=latest
+
+- name: Apply xDebug settings
+  sudo: true
+  lineinfile: dest=/etc/php5/mods-available/xdebug.ini
```

Comment:
Tried to use `ini_file` module for this, but although removing comments is only a problem when user comments something manually here (as there are no comments present at the beginning), there seems to be a known issue with this module unable to edit ini files without any sections. At least I did not find a straight forward way to use the module here. Reverted back to `lineinfile` then.